Sources_v3/Controllers/ChannelController.swift

```diff
@@ -78,6 +78,15 @@ extension Client {
 /// A convenience typealias for `ChannelControllerGeneric` with `DefaultDataTypes`
 public typealias ChannelController = ChannelControllerGeneric<DefaultDataTypes>
 
+/// Describes the flow of the messages in the channel.
+public enum MessageSorting {
+    /// New messages appear on the top of the list.
+    case topToBottom
```

Comment:
Why not a simple `ascending` and `descending`, and we can use the enum everywhere:
```swift
public enum Sorting: Int {
  case ascending = -1, descending = 1
}
```
I have the feeling `topToBottom` and `bottomToTop` is encouraging laziness 😛

Reply:
Laziness? Why? I didn't use `ascending`/`descending` because it can be misleading. Ascending can be understood as "from the oldest message to the latest one" which is not true. We always show latests messages.

I think the naming works good because you don't have to think about it :)

Reply:
There's a certain brain exercise involved in thinking "messages sorted by their date, in ascending order... does that put latest messages on bottom or top?", that's what I meant by "encouraging laziness" :D
I don't think it'd be misleading or vague, I think the confusion you meant is still present: "topToBottom means I'll get older messages?" so a general solution (ascending/descending) would be more useful.

Reply:
OK, what if we renamed `MessageSorting` -> `MessageFlow` ?

Reply:
I want to avoid calling it `sorting` because it's not about sorting.